docs/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
The `github-pages` gem does not support Ruby 3, so I set this to the latest version of v2.

Reply:
Do we have to be specific here, or can we just specify e.g. `2` just to avoid the hassle of bumping this regularly?

Reply:
Unfortunately, this does not support version ranges. So an exact version is always needed.

Reply:
If you reverted to supporting both v2 and v3, should we drop that file as well then so that there's no Ruby version hassle when running docs locally?